docs/developers/docker.rst

```diff
@@ -38,6 +38,22 @@ This builds and runs the docker containers. You can now connect to
 `localhost:28080` to see your Invenio installation. The `admin` user does not
 have any password.
 
+.. note::
+    If you are using `boot2docker`_ you need to set up port forwarding by
+    running the following command in a new terminal:
+
+    .. code-block:: shell
+
+        boot2docker ssh -vnNT \
+            -Llocalhost:28080:localhost:28080 \
```

Comment:
Nope, the `CFG_SITE_URL` is configured to port 28080. The reason for this is that the URL sometimes contains the port and therefor it should be the same in `CFG_SITE_URL` and in the fronted (here: the port forwarding). You can force flask to listen on another port, but because our URL handling is sometimes kinda confusing I decided to use the same internal and external port. See `docker-compose-dev.yml` which uses the same setup.